cmd/helm-operator/main.go

```diff
@@ -112,6 +113,7 @@ func init() {
 
 	gitTimeout = fs.Duration("git-timeout", 20*time.Second, "duration after which git operations time out")
 	gitPollInterval = fs.Duration("git-poll-interval", 5*time.Minute, "period on which to poll git chart sources for changes")
+	gitDefaultChartRef = fs.String("git-default-chart-ref", "master", "ref to clone chart from if ref is unspecified in a HelmRelease")
```

Comment:
I think the length of the flag can be reduced to `--git-default-ref`, as the only Git thing the operator does is syncing charts.